src/images/pages/get-started/mt5/type-of-trades/differs.svg

```diff
@@ -0,0 +1 @@
0-.736-.057c-.774 0-1.282.262-1.523.786V22h-2.046v-7.66h1.933l.057.913c.41-.704.98-1.055 1.706-1.055.227 0 .439.03.637.092l-.028 1.968zm5.233 3.625c0-.25-.124-.447-.372-.591-.248-.144-.645-.273-1.193-.386-1.822-.382-2.733-1.156-2.733-2.322 0-.68.282-1.248.846-1.703.564-.456 1.302-.683 2.213-.683.972 0 1.75.229 2.333.687.583.457.874 1.052.874 1.784h-2.046a.99.99 0 0 0-.283-.726c-.189-.191-.484-.287-.885-.287-.345 0-.611.078-.8.234a.738.738 0 0 0-.283.595c0 .226.107.41.322.548.215.14.577.26 1.087.361.51.102.939.216 1.288.344 1.081.396 1.621 1.083 1.621 2.06 0 .699-.3 1.264-.899 1.696-.6.432-1.373.648-2.322.648-.642 0-1.212-.115-1.71-.344-.498-.229-.888-.543-1.172-.941a2.188 2.188 0 0 1-.424-1.293h1.94c.018.364.153.642.403.836.25.193.585.29 1.006.29.391 0 .687-.074.888-.223.2-.149.301-.343.301-.584z" fill="#231F20"/><path stroke="#293052" stroke-width="3.608" stroke-linecap="round" stroke-linejoin="round" d="M223.127 93.583l-31.861 32.202-42.57-22.314-43.215 24.89-29.81-14.7-36.874-30.686"/><path d="M187.81 144.159h-35.164l-5.545-8.07-5.316 8.07H106.62a4.741 4.741 0 0 0-4.747 4.735v24.512a4.741 4.741 0 0 0 4.747 4.735h81.19a4.753 4.753 0 0 0 3.43-1.346 4.73 4.73 0 0 0 1.43-3.39v-24.473a4.73 4.73 0 0 0-1.416-3.414 4.753 4.753 0 0 0-3.444-1.36z" fill="#E88024"/><path d="M139.24 164.692l2.165-10.38h3.038l-3.532 15.154h-2.848l-2.772-10.002-2.772 10.002h-2.924l-3.57-15.154h3l2.165 10.38 2.848-10.38h2.24l2.963 10.38zm9.874-9.054h-3.038v-2.311h3.038v2.311zm0 13.98h-3.038v-11.366h3.038v11.365zm5.278-11.29v1.629c.357-.547.839-1.001 1.405-1.326a3.804 3.804 0 0 1 1.861-.493c1-.057 1.974.332 2.658 1.061a4.954 4.954 0 0 1 .95 3.372v7.046h-2.772v-7.084a2.27 2.27 0 0 0-.608-1.63 1.825 1.825 0 0 0-1.405-.492 2.36 2.36 0 0 0-1.101.228 2.24 2.24 0 0 0-.798.682v8.183h-3.038v-11.366h2.848v.19zm12.57 6.023h-3.038v-9.925h3.038v9.925zm0 5.228h-3.038v-2.651h3.038v2.651z" fill="#FFF" fill-rule="nonzero"/><ellipse cx="11.468" cy="8.448" rx="8.165" ry="8.373" opacity=".6" transform="translate(211.658 85.135)" fill="#FFF"/><ellipse stroke="#E88024" stroke-width="2.43" cx="223.127" cy="93.583" rx="8.165" ry="8.373"/><path fill="#E88024" d="M246.101 84.756h28.671v18.79h-28.67z"/><path d="M259.848 87.9a5.748 5.748 0 0 1 1.253 0c.39.092.772.218 1.14.379l-.418 1.743-.988-.265h-.987a2.166 2.166 0 0 0-1.747.833 3.328 3.328 0 0 0-.607 2.273c.29-.287.64-.507 1.025-.644.45-.163.927-.24 1.405-.227.97-.042 1.9.38 2.506 1.136.632.839.954 1.87.912 2.917a4.124 4.124 0 0 1-1.14 2.993 3.802 3.802 0 0 1-2.924 1.175 4.03 4.03 0 0 1-3.113-1.288 5.03 5.03 0 0 1-1.178-3.562v-2.348a5.107 5.107 0 0 1 1.367-3.789 4.64 4.64 0 0 1 3.494-1.326zm-.57 5.948a2.74 2.74 0 0 0-1.139.227 1.746 1.746 0 0 0-.721.569v.795c-.051.77.134 1.536.531 2.198.292.473.81.76 1.367.757a1.368 1.368 0 0 0 1.178-.682c.344-.488.53-1.07.531-1.667.03-.578-.13-1.15-.455-1.629a1.444 1.444 0 0 0-1.292-.606v.038z" fill="#FFF" fill-rule="nonzero"/><path fill="#E88024" d="M249.595 94.151l-3.342 3.334-5.202-3.334 5.202-3.296z"/></g></svg>
```

Comment:
if these are exact images that we are using in explanation section there's no need to add them again, we should reuse those

Reply:
i cant see any matches and differs images in explanation section, i added this refering to this card https://trello.com/c/VwyqWeMf

Reply:
as discussed, let's wait until we get the image in all languages before adding it to explanations page.